.jshintrc

```diff
@@ -0,0 +1,32 @@
+{
+  // settings
```

Comment:
I guess github thinks this is a JSON file so these comments are invalid, but it's not a JSON file.